app/assets/javascripts/application.js

```diff
@@ -16,3 +16,7 @@
 //= require jquery-ui
 //= require turbolinks
 //= require_tree .
+//= require js-routes
+
+window.pusher = new Pusher(window.pusher_key);
```

Comment:
I was getting an error in console about the pusher_key not being set right. I switched out `window.pusher_key` to the actual key string and the error went away. not sure where `window.pusher_key` is coming from.